packages/boxel/package.json

```diff
@@ -56,6 +56,7 @@
     "ember-qunit": "^5.1.5",
     "ember-set-body-class": "^1.0.2",
     "ember-svg-jar": "^2.3.3",
+    "ember-template-imports": "^3.0.1",
```

Comment:
Since we're extending the build pipeline of a v1 addon, this needed to be in dependencies, not devDependencies.